<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AltWidget</class>
 <widget class="QWidget" name="AltWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>alt</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>816</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>100</number>
    </property>
    <item row="1" column="2">
     <widget class="QPushButton" name="top_next">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="glasses">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="shoes_next">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="glasses_prev">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="top_prev">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="shoes_prev">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="bot_prev">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="shoes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="belt_prev">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="glasses_next">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="bot">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="belt_next">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="bot_next">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="top">
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="belt">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 25;
outline: none;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="favourites">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
padding: none;
border: transparent solid;
background-color: rgba(255, 255, 0,100);
border-radius: 10%;

}
QPushButton:hover{
background-color: rgba(90,90,90,100);
}</string>
      </property>
      <property name="text">
       <string>Добавить в 
избранное</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="back">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
padding: none;
border: transparent solid;
background-color: rgba(255, 0, 0,100);
border-radius: 10%;

}
QPushButton:hover{
background-color: rgba(90,90,90,100);
}</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
